### ISSUE TITLE
fix: Can't parse html style tag

### ISSUE THREAD

**Issue by cka29** [kingwill101/liquify#11]
```
<style>
  p {
    color: blue;
  } 
</style>
{% if condition %}
  <p> {{ value }} </p>
{% endif %}
```
throws `flutter: parseInput: end of input expected @ 2:5` error and is not rendered. I know we can use `{%raw%}` but it would be nice if single `{` `}` characters were not considered when parsing.

**Comment by kingwill101** [comment]
@cka29 are you on the latest version?
Fixed this issue in #9 where we are failing on single {  tokens

**Comment by cka29** [comment]
@kingwill101 works in latest version, thank you 🙏
